modules/google-speech/src/backend/utils.test.ts

```diff
@@ -1,6 +1,11 @@
 import { closest } from './utils'
 
-const randomNumber = (max = 100) => Math.floor(Math.random() * max)
+const randomNumber = (max = 100) => {
+  const value = Math.floor(Math.random() * max)
+  // Prevents returning 0 because it makes tests fail randomly
+  return value > 1 ? value : randomNumber(max)
```

Comment:
Instead of doing a "potential" infinite loop, we could maybe just add 1 to the result of Math.floor.

Reply:
That's an impossibility